latch_cli/constants.py

```diff
@@ -22,6 +22,7 @@ class LatchConstants:
     base_image: str = (
         "812206152185.dkr.ecr.us-west-2.amazonaws.com/latch-base:fe0b-main"
     )
+    nextflow_latest_version: str = "v1.1.1"
```

Comment:
how does the `latch nextflow upgrade` command work now? 

i don't think it will so we should delete? 

even better if we could have the command edit the Dockerfile to change the version of the base image, but thats more of a nice to have

Reply:
im not worried about backwards compatability here because im p sure the only one that uses that command is Tahir and we can just have him update his Dockerfile

Reply:
done